src/table/paginator.scss

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../icon/index';
+@import './variables';
+
+@mixin iui-paginator {
+  @include iui-reset;
+  @include iui-font-family;
+  display: flex;
+  flex: 1;
+  justify-content: space-around;
+  align-items: center;
+  min-height: $iui-baseline * 5;
+  padding: 0 $iui-m;
+  user-select: none;
+  @include themed {
+    background-color: t(iui-color-background-3);
+  }
+
+  > .iui-left,
+  > .iui-center,
+  > .iui-right {
+    display: flex;
+    align-items: center;
+    position: relative;
+    height: 100%;
+    flex: 1;
+  }
+
+  > .iui-center {
+    width: 33%;
```

Comment:
Don't think this is needed.

Reply:
I think you are right, removed in the next push.